tests/layer_tests/tensorflow_tests/test_tf_TensorScatterUpdate.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import tensorflow as tf
+import pytest
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestTFTensorScatterUpdate(CommonTFLayerTest):
```

Comment:
please run tests locally to check if implementation is correct because I am not sure `ScatterElementsUpdate` is suitable for `TensorScatterUpdate` decomposition

Reply:
@rkazants I'm trying to build but the build is crashing.